addons/packages/contour/1.21.0/package.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: contour.community.tanzu.vmware.com.1.21.0
+spec:
+  refName: contour.community.tanzu.vmware.com
+  version: 1.21.0
+  releaseNotes: "contour 1.21.0 https://github.com/projectcontour/contour/releases/tag/v1.21.0"
+  releasedAt: 2022-02-24T00:00:00Z
+  licenses:
+    - "Apache 2.0"
+  template:
+    spec:
+      fetch:
+        - imgpkgBundle:
+            image: projects.registry.vmware.com/tce/contour@sha256:870ad5b8ef974a0ed78126a2eb0a41e1a8a11413352667a1fd20afe07b0a1ebb
```

Comment:
used image tag `1.21.0_build.1`, not sure if we should document somewhere (besides our internal instructions)

Reply:
now pushed with tag 1.21.0_build.2